src/processing/board_object/CreateOperation.java

```diff
@@ -0,0 +1,18 @@
+package processing.board_object;
+
+/**
+*
+* @author Ahmed Zaheer Dadarkar
+*/
+
+// The Create Operation
+public class CreateOperation implements BoardObjectOperation {
+	
+	// Empty Constructor
```

Comment:
Why do you need an empty constructor unless you are planning to use reflection?

Reply:
Removed empty constructor.